### PR TITLE
Change url for gitmodules

### PR DESCRIPTION
**Pull request by Coenego**

### CODE REVIEW

**Comment by Coenego** [comment]
Assigning to @bp323 for check

**Comment by bp323** [comment]
I think the Travis file can be updated as well as it's currently doing a replace for this.

**Comment by Coenego** [comment]
Back to @bp323

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,6 @@ git:
   submodules: false
 
 before_install:
-  - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
   - git submodule update --init --recursive
```
Comment (bp323) [inline .travis.yml]: Can this be removed as well? Including remvoving the `submodules: false` on the top?

**Comment by Coenego** [comment]
Reverted the Travis file. Let's see how it turns out. Back to @bp323

**Comment by bp323** [comment]
Verified and merged.

**Pull request merged**